### PR TITLE
CORTX-30083:Fix Codacy issues under "net" directory

### PR DESCRIPTION
**Pull request by Vinoth2101**
CORTX-30083: Fix Codacy issues under "net" directory
	- Fixing Codacy issues under net/test dir

Signed-off-by: Vinoth.V <vinoth.v@seagate.com>

# Problem Statement
Codacy error : 
-  "Double quote to prevent globing and words splitting".
-   variable unused. Verify use (or export if used externally).
- %d in format string (no. 2) requires 'int' but the argument type is 'unsigned int'
- Boolean result is used in bitwise operation. Clarify expression with parentheses.

# Design
- We are putting the variable references in double quotes.
- Exported the variables as it is used in different shell scripts
- modified appropriate format string 
- brackets applied , so the Boolean operation happens based on associativity 

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

# Testing Result
Tested with debug message
<< 01net >>
transfer machines endpoint prefix is 0@lo:12345:42
------ test client in user space, test server in user space
--- ping test (test message size is 4KiB)
====CORTX-30083====
====CORTX-30083====
avg out: 20.1MiB/s avg in: 20.1MiB/s total out: 20.1MiB total in: 20.1MiB avg RTT:   1040.351 us
avg out: 19.5MiB/s avg in: 19.5MiB/s total out: 39.0MiB total in: 39.0MiB avg RTT:   1145.171 us
avg out: 18.7MiB/s avg in: 18.7MiB/s total out: 56.2MiB total in: 56.1MiB avg RTT:   1266.184 us
avg out: 18.9MiB/s avg in: 18.9MiB/s total out: 75.8MiB total in: 75.8MiB avg RTT:   1243.033 us
avg out: 19.5MiB/s avg in: 19.5MiB/s total out: 97.9MiB total in: 97.9MiB avg RTT:   1208.526 us
clients total: messages total/failed/bad: sent = 25070/0/0, received = 25065/0/0, bulk sent = 0/0/0, bulk received = 0/0/0, transfers = 25065/0/0; count/min/max/avg/stddev: MPS, sent = 5/4375/5639/5004/461, MPS, received = 5/4375/5636/5003/459, RTT = 25065/137221/8004746/1208479/588752 ns
servers total: messages total/failed/bad: sent = 25076/0/0, received = 25076/0/0, bulk sent = 0/0/0, bulk received = 0/0/0, transfers = 0/0/0; count/min/max/avg/stddev: MPS, sent = 5/4357/5626/5001/460, MPS, received = 5/4365/5628/5002/459, RTT = 0/0/0/0/0 ns
--- bulk test (test message size is 1MiB)
====CORTX-30083====
====CORTX-30083====
avg out:    0B  /s avg in:  464MiB/s total out:  465MiB total in:  465MiB avg RTT:  17025.671 us
avg out:  476MiB/s avg in:  475MiB/s total out:  953MiB total in:  952MiB avg RTT:  16703.677 us
avg out:  463MiB/s avg in:  462MiB/s total out: 1.36GiB total in: 1.36GiB avg RTT:  17181.814 us
avg out:  461MiB/s avg in:  461MiB/s total out: 1.81GiB total in: 1.81GiB avg RTT:  17256.008 us
avg out:  467MiB/s avg in:  467MiB/s total out: 2.29GiB total in: 2.29GiB avg RTT:  17040.427 us
clients total: messages total/failed/bad: sent = 2349/0/0, received = 0/0/0, bulk sent = 2342/0/0, bulk received = 2341/0/0, transfers = 2341/0/0; count/min/max/avg/stddev: MPS, sent = 5/437/490/467/22, MPS, received = 5/437/491/467/22, RTT = 2341/11646241/42280479/17040427/3055819 ns
servers total: messages total/failed/bad: sent = 0/0/0, received = 2350/0/0, bulk sent = 2344/0/0, bulk received = 2344/0/0, transfers = 2344/0/0; count/min/max/avg/stddev: MPS, sent = 5/438/490/467/21, MPS, received = 5/437/490/466/23, RTT = 2344/7158514/34212564/15586631/3285192 ns
net: test status: SUCCESS
20.06user 11.23system 0:34.97elapsed 89%CPU (0avgtext+0avgdata 72008maxresident)k
8inputs+197384outputs (12major+134370minor)pagefaults 0swaps

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: net/ip.c [hunk 1]
```diff
@@ -275,7 +275,7 @@ static int m0_net_ip_lnet_parse(const char *name, struct m0_net_ip_addr *addr)
 	portnum  = tmid | (1 << 10) | ((portal - 30) << 11);
 	if (portnum > M0_NET_IP_PORT_MAX)
 		return M0_ERR(-EINVAL);
-	snprintf(port, ARRAY_SIZE(port), "%d", (uint32_t)portnum);
+	snprintf(port, ARRAY_SIZE(port), "%u", (uint32_t)portnum);
```
Comment (nkommuri) [inline net/ip.c:278]: portnum is integer, not unsigned int. then why %u ?

File: net/net.c [hunk 2]
```diff
@@ -114,8 +111,8 @@ M0_INTERNAL bool m0_net_endpoint_is_valid(const char *endpoint)
 		return M0_RC(false);
 	endpoint += strlen(networks[i]);
 
-	if (m0_streq(networks[i], "@lo")) {
-		if (!m0_streq(addr, "0"))
+	if (streq(networks[i], "@lo")) {
+		if (!streq(addr, "0"))
```
Comment (nkommuri) [inline net/net.c:115]: Is streq() a library call? where is it defined?

**Comment by rkothiya** [comment]
<h1>Jenkins CI Result : <a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642/>Motr#1642</a></h1><h2>Motr Test Summary</h2><table><tr><th>Test Result</th><th>Count</th><th>Info</tr><tr><td>:x:Failed</td><td>2</td><td><details><summary>:file_folder:</summary><p><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/49motr-rpc-cancel target=_blank>04motr-single-node/49motr-rpc-cancel</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/00userspace-tests target=_blank>01motr-single-node/00userspace-tests</a></p></details></td></tr><tr><td>:checkered_flag:Skipped</td><td>32</td><td><details><summary>:file_folder:</summary><p><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/28sys-kvs target=_blank>01motr-single-node/28sys-kvs</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/35m0singlenode target=_blank>01motr-single-node/35m0singlenode</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/04initscripts target=_blank>01motr-single-node/04initscripts</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/37protocol target=_blank>01motr-single-node/37protocol</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/51kem target=_blank>02motr-single-node/51kem</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/20rpc-session-cancel target=_blank>02motr-single-node/20rpc-session-cancel</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/10pver-assign target=_blank>02motr-single-node/10pver-assign</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/21fsync-single-node target=_blank>02motr-single-node/21fsync-single-node</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/13dgmode-io target=_blank>02motr-single-node/13dgmode-io</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/14poolmach target=_blank>02motr-single-node/14poolmach</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/11m0t1fs target=_blank>02motr-single-node/11m0t1fs</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/26motr-user-kernel-tests target=_blank>02motr-single-node/26motr-user-kernel-tests</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/08spiel target=_blank>02motr-single-node/08spiel</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/06conf target=_blank>03motr-single-node/06conf</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/36spare-reservation target=_blank>03motr-single-node/36spare-reservation</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/34sns-repair-1n-1f target=_blank>04motr-single-node/34sns-repair-1n-1f</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/08spiel-sns-repair-quiesce target=_blank>04motr-single-node/08spiel-sns-repair-quiesce</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/28sys-kvs-kernel target=_blank>04motr-single-node/28sys-kvs-kernel</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/11m0t1fs-rconfc-fail target=_blank>04motr-single-node/11m0t1fs-rconfc-fail</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/08spiel-sns-repair target=_blank>04motr-single-node/08spiel-sns-repair</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/19sns-repair-abort target=_blank>04motr-single-node/19sns-repair-abort</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/22sns-repair-ios-fail target=_blank>04motr-single-node/22sns-repair-ios-fail</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/18sns-repair-quiesce target=_blank>05motr-single-node/18sns-repair-quiesce</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/12fwait target=_blank>05motr-single-node/12fwait</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/16sns-repair-multi target=_blank>05motr-single-node/16sns-repair-multi</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/07mount-fail target=_blank>05motr-single-node/07mount-fail</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/15sns-repair-single target=_blank>05motr-single-node/15sns-repair-single</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/23sns-abort-quiesce target=_blank>05motr-single-node/23sns-abort-quiesce</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/17sns-repair-concurrent-io target=_blank>05motr-single-node/17sns-repair-concurrent-io</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/07mount target=_blank>05motr-single-node/07mount</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/07mount-multiple target=_blank>05motr-single-node/07mount-multiple</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/12fsync target=_blank>05motr-single-node/12fsync</a></p></details></td></tr><tr><td>:heavy_check_mark:Passed</td><td>43</td><td><details><summary>:file_folder:</summary><p><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/43m0crate target=_blank>01motr-single-node/43m0crate</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/05confgen target=_blank>01motr-single-node/05confgen</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/06hagen target=_blank>01motr-single-node/06hagen</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/52motr-singlenode-sanity target=_blank>01motr-single-node/52motr-singlenode-sanity</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/01net target=_blank>01motr-single-node/01net</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/01kernel-tests target=_blank>01motr-single-node/01kernel-tests</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/03console target=_blank>01motr-single-node/03console</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/01motr-single-node/02rpcping target=_blank>01motr-single-node/02rpcping</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/07m0d-fatal target=_blank>02motr-single-node/07m0d-fatal</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/67fdmi-plugin-multi-filters target=_blank>02motr-single-node/67fdmi-plugin-multi-filters</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/53clusterusage-alert target=_blank>02motr-single-node/53clusterusage-alert</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/02motr-single-node/41motr-conf-update target=_blank>02motr-single-node/41motr-conf-update</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/61sns-repair-motr-1n-1f target=_blank>03motr-single-node/61sns-repair-motr-1n-1f</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/72spiel-sns-motr-repair-quiesce target=_blank>03motr-single-node/72spiel-sns-motr-repair-quiesce</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/08spiel-multi-confd target=_blank>03motr-single-node/08spiel-multi-confd</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/69sns-repair-motr-quiesce target=_blank>03motr-single-node/69sns-repair-motr-quiesce</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/62sns-repair-motr-mf target=_blank>03motr-single-node/62sns-repair-motr-mf</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/70sns-failure-after-repair-quiesce target=_blank>03motr-single-node/70sns-failure-after-repair-quiesce</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/63sns-repair-motr-1k-1f target=_blank>03motr-single-node/63sns-repair-motr-1k-1f</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/60sns-repair-motr-1f target=_blank>03motr-single-node/60sns-repair-motr-1f</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/66sns-repair-motr-abort-quiesce target=_blank>03motr-single-node/66sns-repair-motr-abort-quiesce</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/24motr-dix-repair-lookup-insert-spiel target=_blank>03motr-single-node/24motr-dix-repair-lookup-insert-spiel</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/68sns-repair-motr-shutdown target=_blank>03motr-single-node/68sns-repair-motr-shutdown</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/64sns-repair-motr-ios-fail target=_blank>03motr-single-node/64sns-repair-motr-ios-fail</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/71spiel-sns-motr-repair target=_blank>03motr-single-node/71spiel-sns-motr-repair</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/24motr-dix-repair-lookup-insert-m0repair target=_blank>03motr-single-node/24motr-dix-repair-lookup-insert-m0repair</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/04sss target=_blank>03motr-single-node/04sss</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/03motr-single-node/65sns-repair-motr-abort target=_blank>03motr-single-node/65sns-repair-motr-abort</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/73motr-io-small-disks target=_blank>04motr-single-node/73motr-io-small-disks</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/48motr-raid0-io target=_blank>04motr-single-node/48motr-raid0-io</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/74motr-di-corruption-detection target=_blank>04motr-single-node/74motr-di-corruption-detection</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/25m0kv target=_blank>04motr-single-node/25m0kv</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/44motr-rm-lock-cc-io target=_blank>04motr-single-node/44motr-rm-lock-cc-io</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/04motr-single-node/45motr-rmw target=_blank>04motr-single-node/45motr-rmw</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/23dix-repair-m0repair target=_blank>05motr-single-node/23dix-repair-m0repair</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/43motr-sync-replication target=_blank>05motr-single-node/43motr-sync-replication</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/42motr-utils target=_blank>05motr-single-node/42motr-utils</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/45motr-sns-repair-N-1 target=_blank>05motr-single-node/45motr-sns-repair-N-1</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/40motr-dgmode target=_blank>05motr-single-node/40motr-dgmode</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/23dix-repair-quiesce-m0repair target=_blank>05motr-single-node/23dix-repair-quiesce-m0repair</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/23spiel-dix-repair-quiesce target=_blank>05motr-single-node/23spiel-dix-repair-quiesce</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/44motr-sns-repair target=_blank>05motr-single-node/44motr-sns-repair</a></br><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit/05motr-single-node/23spiel-dix-repair target=_blank>05motr-single-node/23spiel-dix-repair</a></p></details></td></tr><tr><td>Total</td><td>77</td><td><a href=https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1642//testReport/junit>:link:</a></td></tr></table><h2> CppCheck Summary</h2><p> &emsp;&emsp;&emsp;Cppcheck: No new warnings found :thumbsup:</p>

**Review by nkommuri — APPROVED** [review]

**Comment by rkothiya** [comment]
Currently Memory leak pre-merge job is failing for all the pr's and hence we are ignoring it, as we do not want to block merge. Once we streamline that we will mandate its passing.

**Pull request merged**